lib/common/options.c

```diff
@@ -337,7 +355,7 @@ pcmk__set_env_option(const char *option, const char *value)
  * or a list of daemon names, return true if the option is enabled for a given
  * daemon.
  *
- * \param[in] daemon   Daemon name
+ * \param[in] daemon   Daemon name (can be NULL)
```

Comment:
A NULL `daemon` is currently allowed, even though we're not calling this with a NULL `daemon` now. If `daemon` is NULL and `value` is true, we still return true. So might as well document it.

But we do need to NULL-check `daemon` before passing it to `strstr()`.